src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,65 @@
+$blockSize: 100px;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $blockSize * 7 + 10px;
+
+  &--start-day {
+    $daysOfWeeks: mon, tue, wed, thu, fri, sat, sun;
+    $indexOfWeek: 0;
+
+    @each $day in $daysOfWeeks {
```

Comment:
create `functions.scss ` for your functions

Reply:
done